Resources/views/Collector/db.html.twig

```diff
@@ -98,289 +107,340 @@
         .sql-runnable.sf-toggle-content.sf-toggle-visible { display: flex; flex-direction: column; }
         .sql-runnable button { align-self: end; }
+        table .sort-icon { display: inline-flex; vertical-align: sub; }
+        table .sort-icon svg { height: 20px; width: 20px; margin-left: 2px; stroke-width: 2.5; color: var(--color-link); }
     </style>
 
-    <h2>Query Metrics</h2>
+    <h2>Doctrine</h2>
 
     <div class="metrics">
-        <div class="metric">
-            <span class="value">{{ collector.querycount }}</span>
-            <span class="label">Database Queries</span>
-        </div>
-
-        <div class="metric">
-            <span class="value">{{ collector.groupedQueryCount }}</span>
-            <span class="label">Different statements</span>
-        </div>
+        <div class="metric-group">
+            <div class="metric">
+                <span class="value">{{ collector.querycount }}</span>
+                <span class="label">Queries</span>
+            </div>
 
-        <div class="metric">
-            <span class="value">{{ '%0.2f'|format(collector.time * 1000) }} ms</span>
-            <span class="label">Query time</span>
-        </div>
+            <div class="metric">
+                <span class="value">{{ collector.groupedQueryCount }}</span>
+                <span class="label">Different statements</span>
+            </div>
 
-        <div class="metric">
-            <span class="value">{{ collector.invalidEntityCount }}</span>
-            <span class="label">Invalid entities</span>
+            <div class="metric">
+                <span class="value">{{ '%0.2f'|format(collector.time * 1000) }} <span class="unit">ms</span></span>
+                <span class="label">Query time</span>
+            </div>
         </div>
 
         {% if collector.cacheEnabled %}
-            <div class="metric">
-                <span class="value">{{ collector.cacheHitsCount }}</span>
-                <span class="label">Cache hits</span>
-            </div>
-            <div class="metric">
-                <span class="value">{{ collector.cacheMissesCount }}</span>
-                <span class="label">Cache misses</span>
+            <div class="metric-divider"></div>
+            <div class="metric-group">
+                <div class="metric">
+                    <span class="value">{{ collector.cacheHitsCount }}</span>
+                    <span class="label">Cache hits</span>
+                </div>
+                <div class="metric">
+                    <span class="value">{{ collector.cacheMissesCount }}</span>
+                    <span class="label">Cache misses</span>
+                </div>
+                <div class="metric">
+                    <span class="value">{{ collector.cachePutsCount }}</span>
+                    <span class="label">Cache puts</span>
+                </div>
             </div>
+        {% endif %}
+
+        {% if collector.invalidEntityCount %}
```

Comment:
This also seem quite arbitrary decision to me to hide the section when count is 0

Reply:
Agreed, got rid of this.